css/index.css

```diff
@@ -3,144 +3,308 @@
 }
 
 .top-content {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-bottom: 20px;
+  margin-top: 5%;
+  border-bottom: 1px dashed black;
 }
 
 .top-content .text-container {
-    width: 48%;
-    padding: 0 1%;
-    padding-bottom: 20px;  
+  width: 48%;
+  padding: 0 1%;
+  padding-bottom: 20px;
+  margin-top: 5%;
 }
 
 .middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  margin-bottom: 20px;
+  border-bottom: 1px dashed black;
 }
 
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+  padding: 0 2%;
+  margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  width: 12.5%;
+  height: 100px;
+  background: black;
+  margin: 20px 2.5%;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
 .bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+  display: flex;
+  margin: 0 2% 20px;
+  justify-content: space-around;
 }
 
 .bottom-content .text-container {
-    padding-right: 4%;
+  padding-right: 4%;
 }
 
 .bottom-content .text-container:last-child {
-    padding-right: 0;
+  padding-right: 0;
 }
 
 footer {
-    width: 100%;
-    background: black;
+  width: 100%;
+  background: black;
 }
 
 footer nav {
-    width: 60%;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 2%;
+  font-size: 14px;
 }
 
 footer nav a {
-    color: white;
-    text-decoration: none;
-}
\ No newline at end of file
+  color: white;
+  text-decoration: none;
+}
+
+.headLinks {
+  display: flex;
+  justify-content: space-evenly;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+.headLinks a {
+  margin-top: 2.5%;
+}
+.boxes:nth-child() {
+  background-color: blueviolet;
+}
+/* ABOUT.HTML CSS*/
+.about {
+  margin-top: 2%;
+  margin-bottom: 5%;
+  border-bottom: 0.1rem dotted black;
+  padding: 3%;
+}
+.learnButton {
+  border: 0.2rem solid black;
+  border-radius: 20%;
+  padding: 2%;
+}
+.flex-box {
+  display: flex; /* I used this property because it was the most quick and simple way to create a grid*/
+  flex-wrap: wrap;
+}
+.row {
+  flex: 1;
+  border-bottom: 0.1rem dotted black;
+  padding-bottom: 5%;
+  margin-bottom: 5%;
+  margin-right: 1.5%;
+}
+.row p {
+  margin-bottom: 5%;
+}
+.aboutFooter {
+  margin-top: 2rem;
+}
```

Comment:
Good job allowing the parent to style most of the CSS styling and only overriding the color for each individual box.